docs/types.md

```diff
@@ -14,7 +14,7 @@ Tag | Type | Description | Example
 [<span class="tag col"></span>](types.md)&nbsp;| color | See section [Color](#color) for more info. |
 [<span class="tag fun"></span>](types.md)&nbsp;| function | See section [Function](#function) for more info. |
 [<span class="tag obj"></span>](types.md)&nbsp;| object | An in-game physical Object. Sometimes Global. | `#!lua Global or self`
-[<span class="tag pla"></span>](types.md)&nbsp;| player | An in-game Player. | `#!lua Player["White"]`
+[<span class="tag pla"></span>](types.md)&nbsp;| player | An in-game Player. See [Player](player.md) for more info. | `#!lua Player["White"]`
```

Comment:
"See section Player" for consistency

Reply:
I chose against "section Player" because it is not a section of the types page. Maybe I should have used "the Player page". I'm not sure.